hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -693,6 +694,14 @@ message SwitchExceedThrottleQuotaResponse {
   required bool previous_exceed_throttle_quota_enabled = 1;
 }
 
+message BalancerDecisionRequest {
+  optional uint32 limit = 5 [default = 250];
```

Comment:
This default seems low, and should be an interface method param option and a class file constant, not a proto constant, IMHO.

The limit should be generic to the getList() interface and not specific to the log entry type. 

The default should align with the server side default ring buffer size.  The idea is the client will get all of the entries by default, but can ask for fewer depending on use case or constraints.